extensions/amp-access-fewcents/0.1/fewcents-impl.js

```diff
@@ -93,9 +139,40 @@ export class AmpAccessFewcents {
    * @private
    */
   prepareAuthorizeUrl_() {
-    // [TODO]: Actual implementation
-    dev().fine(TAG, 'Publishers config', this.fewcentsConfig_);
-    return;
+    const accessKey = this.fewcentsConfig_['accessKey'];
+    const articleIdentifier = this.fewcentsConfig_['articleIdentifier'];
+    const category = this.fewcentsConfig_['category'];
+
+    const {hostname} = parseUrlDeprecated();
+
+    return (
+      CONFIG_BASE_PATH +
```

Comment:
Is everything properly escaped (especially URL arguments)?

Reply:
Fixed.